people/service.go

```diff
@@ -23,6 +23,7 @@ type PeopleService interface {
 	getPersonByUUID(uuid string) (person, bool, error)
 	getCount() (int, error)
 	isInitialised() bool
+	loadDB() error
 	Shutdown() error
 }
 
```

Comment:
I can't comment it on the spot, but I think it would be cleaner also in `NewPeopleService` to don't pass the service as an argument to the anonymous go function